LTI 1.3 instance selection implementation adjustment

miguelaenlle opened the pull request:
Related to [this discussion](https://github.com/PrairieLearn/PrairieLearn/pull/11209#discussion_r1931333974) in #11209 .

- Improved the implementation of LTI 1.3 instance selection by moving `<script>` logic to a separate `client` file and stashing the prefix in a `data-url-prefix` attribute.

nwalters512 commented on apps/prairielearn/assets/scripts/instructorInstanceAdminLti13Client.ts:
```diff
@@ -0,0 +1,16 @@
+  if (!prefix) return;
+
+  selectLti13Instance?.addEventListener('change', () => {
+    const selectedOption = selectLti13Instance?.querySelector<HTMLOptionElement>('option:selected');
```
`option:selected` isn't a valid selector. I'd recommend using the [`selectedOptions`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/selectedOptions) property on `selectLti13Instance` instead.

nwalters512 reviewed (commented):
The main issue here is the use of `options:selected`, which doesn't work. That'll need to be fixed one way or another before this is merged.

jonatanschroeder commented:
Would it make more sense to replace this select with a bootstrap dropdown containing links to the different instances?

nwalters512 commented:
> Would it make more sense to replace this select with a bootstrap dropdown containing links to the different instances?

I like this idea. That's probably more accessible too, as they'll be announced as links. Probably worth prototyping!

miguelaenlle commented:
> Would it make more sense to replace this select with a bootstrap dropdown containing links to the different instances?

@jonatanschroeder @nwalters512 , I implemented this idea, and removed `instructorInstanceAdminLti13Client.ts` as it's no longer needed with this implementation. Let me know your thoughts on this version.

nwalters512 commented:
@miguelaenlle I like it! Can we use the "active" stuff from these docs to indicate the active instance both visually and to screen readers? https://getbootstrap.com/docs/5.3/components/dropdowns/#active

miguelaenlle commented:
> Added active class to the dropdown item

Awesome! I adjusted the `dropdown-item`s such that the selected one receives the `active` class.

nwalters512 commented:
One more thing, the above Bootstrap docs link explained using `aria-current="page"` to help folks who are using assistive technologies. Can you add that as well?

miguelaenlle commented:
> One more thing, the above Bootstrap docs link explained using `aria-current="page"` to help folks who are using assistive technologies. Can you add that as well?

I added this change 👍 

https://github.com/PrairieLearn/PrairieLearn/blob/9d213211631f5064eb6ff54f655730167b407cdf/apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.html.ts#L78-L92
